internal/api/otp.go

```diff
@@ -83,6 +88,14 @@ func (a *API) SmsOtp(w http.ResponseWriter, r *http.Request) error {
 	if err := json.Unmarshal(body, params); err != nil {
 		return badRequestError("Could not read sms otp params: %v", err)
 	}
+	// For backwards compatibility, we default to SMS if params Channel is not specified
+	if params.Phone != "" && params.Channel == "" {
+		params.Channel = sms_provider.SMSProvider
+	}
+	if params.Phone != "" && !sms_provider.IsValidMessageChannel(params.Channel, *config) {
+		return badRequestError("Invalid Channel. Please use 'sms' or 'whatsapp'")
```

Comment:
I would maybe word this as `Invalid channel, supported values are 'sms' or 'whatsapp'`